java-maps-fleetengine-delivery/google-maps-fleetengine-delivery/pom.xml

```diff
@@ -1,116 +0,0 @@
-<?xml version='1.0' encoding='UTF-8'?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.google.maps</groupId>
-  <artifactId>google-maps-fleetengine-delivery</artifactId>
```

Comment:
This hasn't been published. Good.

![image](https://github.com/googleapis/google-cloud-java/assets/28604/580018b2-809e-42b1-a384-9699720b7682)